unfetter-discover/unfetter-ui
# Angular upgrade

Angular 5, Material, and CLI upgrades

## To Test
```
cd unfetter-discover/unfetter-ui
npm run lint
npm run build:dev
npm run analyze:dev
npm run serve:local:dev:demo
npm run build:prod
npm run coverage
npm run test
```

click around the app, make sure everything looks ok
